### DISCUSSION TITLE
8번째 미팅 때 논의하고 싶은 내용

### DISCUSSION THREAD

**Discussion by Joseph-Cha** [Joseph-Cha/JustFight_Client#34] [General]
1. 일단 약 6주 동안 기획, 요구사항 분석, 설계, 구현, 테스트 이렇게 한 싸이클을 돌리고 느낀 후기
2. 향후 진행 방향

**Comment by jongfeel** [comment]
테스트 코드 작성하면서 느낀 점
문제 풀어 보면서 느껴봤던 설계 방향성

향후 진행 방향:
- 조금 많은 양의 설계 리뷰 => 설계 반영한 코딩 + 테스트 코드
- 설계 branch, 구현+테스트 branch 별도로 pull request 진행
